Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,31 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            if (array == null)
+            {
+                return;
+            }
+
+            var flag = true;
+            var temp = 0;
```

Comment:
You don't need temp here. You only use it in line 30 and line 32. :)

Reply:
I need it because otherwise the sort routine wil break

Reply:
It won't. 
```
var temp = array[j];
array[j] = array[j + 1];
array[j + 1] = temp;
```

Reply:
changed